scripts/profile_startup.py

```
@@ -0,0 +1,242 @@
+        if not os.path.exists(f):
+            print(f"Error: File not found: {f}")
+            sys.exit(1)
+
+    raven_path = os.path.expanduser("~/repos/raven/target/release/raven")
+
+    # Start Raven with perf logging
+    env = os.environ.copy()
+    env["RAVEN_PERF"] = "1"
+    env["RUST_LOG"] = "raven=info"
+
+    print(f"Starting Raven LSP server...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)